app/services/eligible_scopes_service.rb

```diff
@@ -55,7 +55,7 @@ def service_level_config
   end
 
   def innermost_config
-    raise "Invalid Client Involvement Type '#{@client_involvement_type}" unless %w[A D I W Z].freeze.include?(@client_involvement_type)
+    raise "Invalid Client Involvement Type '#{@client_involvement_type}'" unless %w[A D I W Z].freeze.include?(@client_involvement_type)
```

Comment:
I was going to say is it worth using the client_involvement_type model to query the all ccms_codes, then realise we do not have one.

**Not for now, or not a blocker** but is it worth adding an `.all|defaults` class method that returns the private `default_response` method result and using here

And then we could
```suggestion
    raise "Invalid Client Involvement Type '#{@client_involvement_type}'" unless ClientInvolvementTypeService.all.pluck(:ccms_code).include?(@client_involvement_type)
```